doc/tutorial.rst

```diff
@@ -0,0 +1,131 @@
+Tutorial
+========
+
+This should get you started with using ``dnaio``.
+The only essential concepts to know about are
+the `dnaio.open` function and the `SequenceRecord` object.
+
+
+Reading
+-------
+
+The main interface for reading and writing sequence files is the `dnaio.open` function.
+For example, this program reads in a FASTQ file and computes the total number of nucleotides
+it contains::
+
+    import dnaio
+
+    with dnaio.open("reads.fastq.gz") as reader:
+        bp = 0
+        for record in reader:
+            bp += len(record)
+    print(f"The input file contains {bp/1E6:.1f} Mbp")
+
+As can be seen from the ``.gz`` file extension,
+the input file is gzip-compressed.
+`dnaio.open` detects and handles this automatically by opening the file with
+`xopen <https://github.com/pycompression/xopen/>`_.
+
+Here, the call to `dnaio.open` returns a `FastqReader` object.
+Iterating over it in the ``for`` loop results in `SequenceRecord` objects.
```

Comment:
On the next line double backticks are used for SequenceRecord. Was this intentional?

Reply:
I wanted to create a link only for the first occurrence to avoid too much blue in the text, so this is intentional. (Apart from the fact that it doesn’t work as I discovered now.)